COMPATIBILITY.md

```diff
@@ -0,0 +1,40 @@
+# Instrumented Library Compatibility
+
+Instrumentation is provided for the following Go packages.
+
+- [database/sql](#database/sql)
+- [github.com/segmentio/kafka-go](#github.com/segmentio/kafka-go)
+- [google.golang.org/grpc](#google.golang.org/grpc)
+- [net/http](#net/http)
+
+## database/sql
+
+[Package documentation](https://pkg.go.dev/database/sql)
+
+Supported version ranges: 
+
+- 1.12.0 to 1.22.5
```

Comment:
I propose to use the same naming as in https://pkg.go.dev/database/sql?tab=versions

```suggestion
- `go1.12` to `go1.22.5`
```